package-lock.json

```diff
@@ -5340,6 +5339,8 @@
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8=",
       "dev": true
     },
+<<<<<<< HEAD
```

Comment:
Lots of merge conflicts here, please double-check

/cc @kevinlee12 maybe we should have a lint check for ensuring no "<<<" or ">>>" or "HEAD" in any files? I think it's common for merge conflicts to get resolved wrongly.

Reply:
We actually do have a lint check for that, I've filed https://github.com/oppia/oppia/issues/7321 to investigate why it didn't pick this up.

Reply:
Sorry, that is my fault, I used `git push --no-verify` because I needed to push quickly since I need to work on a different computer.

Reply:
But the lint tests should be failing anyway.

Reply:
Yup, that's why I've filed that issue :)